src/GrandDesignsHeader.js

```diff
@@ -0,0 +1,12 @@
+import gd from "./gd.png";
+import './App.css';
+
+function GrandDesignsHeader () {
+    return (
+        <div className="Gd-header">
+        <img src={gd} className="Gd-logo" alt="grand-designs-logo" />
```

Comment:
A small point but correct indentation for HTML works wonders as projects grow: 
```
<div>
    <img/>
</div>
```

Out of interest, which IDE are you using for doing this? If you're using VSCode, I'd suggest looking at installing the [Prettier plugin](https://prettier.io/). It's become an industry standard in its own right because of its auto-formatting features - it saves soooo much time!

Reply:
I'm using Visual Studio and have just installed the Prettier plugin! Cheers!